x/evm/keeper/context_stack.go

```diff
@@ -0,0 +1,76 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type cachedContext struct {
```

Comment:
we should add a comment that the fields are the result values from `ctx.CacheCtx()`

Reply:
done